### DISCUSSION TITLE
Import of old data

### DISCUSSION THREAD

**Discussion by tcurdt** [plausible/analytics#564] [Anything Else]
Is there a way to import historic data? From

- Google Ananlytics
- Nginx Server Logs

If not that would be a nice thing to have to make the switch easier.

**Comment by metmarkosaric** [comment]
thanks @tcurdt! not possible at this time unfortunately but we'd definitely like to support both. see #211

**Comment by metmarkosaric** [comment]
Universal Analytics, the current version of Google Analytics, [will be sunset](https://plausible.io/blog/universal-google-analytics-is-dead) and will stop counting stats on July 1st 2023. There’s no way to import your historical Universal Analytics stats into the new Google Analytics 4, so we've put a lot of effort into making a Google Analytics stats import tool ourselves. 

You can now import your historical Google Analytics (Universal Analytics) stats into your Plausible dashboard. You can import the stats from your first Google Analytics visitor until your first Plausible Analytics visitor. [See more here](https://plausible.io/docs/google-analytics-import).
